src/Authoring/WinRT.SourceGenerator/WinRTRules.cs

```diff
@@ -21,121 +21,130 @@ private static DiagnosticDescriptor MakeRule(string id, string title, string mes
                 helpLinkUri: "https://docs.microsoft.com/en-us/previous-versions/hh977010(v=vs.110)");
         }
 
+        public static DiagnosticDescriptor ImplementerUseSameVariableName = MakeRule(
+            "CsWinRT1113", 
+            "Class implementaiton method should use same paramter as interface method",
+            "Class '{0}' implements method '{1}' with parameter '{2}', but '{1}' was declared in interface '{3}' with parameter '{4}'." 
+            + "Parameter names must match exactly in the Windows Runtime.");
+        
         public static DiagnosticDescriptor PrivateGetterRule = MakeRule(
-            "WME", 
+            "CsWinRT", 
```

Comment:
Is there no # for this.

Reply:
I see a couple actually without a #.  Do we want them to all have # or no?

Reply:
Yeah when we make the docs page I'll have a table of all of them. Was going to assign the number then, can do it now though. I can come up with numbers that aren't the ones we have already -- there's even a couple like 1060(a) that might just be clearer under a new number.